src/current/v23.2/sso-db-console.md

```diff
@@ -66,13 +66,25 @@ You must have the ability to update your cluster settings, which you can achieve
 
 You must configure the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) in the following table to enable SSO authentication to the DB Console. Refer to the [Update your cluster settings](#update-your-cluster-settings) section to configure your cluster settings.
 
-| Cluster Setting | Description 
+{% capture oidc_authentication_redirect_url_json %}<br /><br />
+
+{% capture json %}{% include_cached copy-clipboard.html %}
+~~~ json
+{
+  "redirect_urls": {
+    "us-east-1": "https://{cluster_ip_address}:8080/oidc/v1/callback",
+    "eu-west-1": "example.com"
+  }
+}
+{% endcapture %}{{ json }}~~~{% endcapture %}
+
+| Cluster Setting | Description
```

Comment:
There is an issue with rendering the table here, from line 86, looks like it is something to do with {{ oidc_authentication_redirect_url_json }}. Ref: https://deploy-preview-17996--cockroachdb-docs.netlify.app/docs/v23.2/sso-db-console.html#cluster-settings

Reply:
Fixed.